docs/verifier/query-builder.md

```diff
@@ -2,59 +2,209 @@
+
+- **Equal:** Checks if the value is equal to the specified value.
+- **Not Equal:** Verifies if the value is not equal to the specified value.
+- **Less Than:** Assesses if the value is less than the given value.
+- **Greater Than:** Evaluates if the value is greater than the specified value.
+- **Matches one of the values:** Determines if the value is within a given set of values.
+- **Matches none of the values:** Checks if the value is not within a specified set.
 
-Then, one can copy and paste the snippet on the verifier API `GetAuthRequest()` function.
+Choose the appropriate operator based on the condition you need to validate.
 
-:::info
+### Attribute value
 
-The idea behind the Query Builder is to facilitate the query elaboration process by generating the whole section relative to "Request for specific proof" of the auth request endpoint code, as it can be seen on the [<ins>Run a Verifier tutorial</ins>](/docs/verifier/verification-library/verifier-set-up.md#verifier-server-setup), more specifically on point 3.
+The "Attribute value" field allows users to input the specific value or range of values to be compared with the selected attribute in the prover's credential. This field adapts to the type of the chosen attribute, accommodating various data types such as strings, numbers, booleans, dates, and more. This ensures that the value provided is compatible with the selected operator and the nature of the attribute.
+
+### Issuer DID
+
+This field in the query form is used to specify the Decentralized Identifier (DID) of the issuer that is trusted by the verifier. By entering a specific DID, the verifier can confirm that the credential presented by the prover was indeed issued by the specified issuer.
+
+:::caution
+
+Using `*` as a wildcard to accept credentials from any issuer is possible but not recommended, particularly in production environments. This approach may compromise the query's security and integrity, as it will validate credentials issued by any issuer.
+
+:::
+
+:::caution
+
+Currently, the system supports specifying only a single DID. This means users have the choice to either provide one specific issuer DID or use `*` to allow any issuer. However, the former is strongly advised for maintaining robust verification standards.
 
 :::
+
+### Skip Revocation Check
+
+This checkbox, when enabled, instructs the circuit to bypass the verification step that checks whether a credential has been revoked.
+
+:::caution
+
+Opting to skip the revocation check is generally not advised. This check is a crucial security feature, ensuring that the credential being used is still valid and has not been rescinded by the issuer. Skipping this step can lead to the acceptance of outdated or invalidated credentials, potentially compromising the integrity of the verification process and the trustworthiness of the results.
+
+:::
+
+## Verifying a query
+
```

Comment:
> After setting up the query in the Query Builder, the verification can be performed (which differs for off-chain and on-chain queries).

Maybe it's worth adding that the verification screen you'll see depends on the `Circuit ID` field of the query builder form?